.devcontainer/devcontainer.json

```
@@ -12,8 +15,8 @@
                 "editor.detectIndentation": false,
                 "r.lsp.diagnostics": false,
                 "r.plot.useHttpgd": true,
-                "r.rpath.linux": "/usr/bin/R",
-                "r.rterm.linux": "/usr/bin/R",
+                "r.rterm.linux": "/workspaces/r-dev-env/scripts/launch_r.sh",
+                "r.rpath.linux": "/workspaces/r-dev-env/build/r-devel/bin/R",
```

`r.rpath.linux` should be `/usr/bin/R` here as this is the only version of R that exists to begin with.
```suggestion
                "r.rterm.linux": "/workspaces/r-dev-env/scripts/launch_r.sh",
                "r.rpath.linux": "/usr/bin/R",
```